spec/models/period_contribution_calculator_spec.rb

```diff
@@ -108,5 +108,43 @@
         expect(period_contribution.tax_relief).to eq(1.72)
       end
     end
+
+    context 'when employee is contributing more than 40,000 per year' do
+      let(:eligible_salary) { 80_000 }
+      let(:employee_percent) { 60 }
+      let(:employer_percent) { 1 }
+
+      context 'when yearly frequency' do
+        let(:salary_frequency) { 1 }
```

Comment:
How about using named constants for the salary frequency so that the can be understood at first sight?

`let(:salary_frequency) { YEARLY }`
`let(:salary_frequency) { MONTHLY }`
`let(:salary_frequency) { FORTNIGHTLY }`
`let(:salary_frequency) { WEEKLY }`